tpls/appcache-frame.tpl

```diff
@@ -90,9 +90,6 @@
         clearInterval(downloadingInterval);
         downloadingInterval = null;
       }
-
-      applicationCache.removeEventListener('downloading', onDownloadingEvent);
-      applicationCache.removeEventListener('progress', onDownloadingEvent);
```

Comment:
Also, I'm probably wrong about firing it multiple times because of `updatingFired` check. But this check will prevent `onUpdating` event to fire when `autoUpdate` happens. In other words, it's more complicated than just removing those lines.